futbin-club-worth.user.js

```diff
@@ -13,51 +13,61 @@
 // @supportURL  https://github.com/Mardaneus86/futwebapp-tampermonkey/issues
 // ==/UserScript==
 // ==OpenUserJS==
-// @author Mardaneus86
+// @author Mardaneus86, Kyderman
 // ==/OpenUserJS==
 (function () {
-  'use strict';
+    'use strict';
 
-  gNavManager.onScreenRequest.observe(this, function(obs, event) {
-    if(event === "MyClubSearch") {
-      var gettingPlayerData = setInterval(function() {
-        var playersInClub = repositories.Item._club._players._collection;
-
-        if (!playersInClub || Object.keys(playersInClub).length === 0) {
-          return;
-        }
-        clearInterval(gettingPlayerData);
-
-        var playerIds = Object.keys(playersInClub).map(function(key) { return playersInClub[key]._metaData.id});
-        var futbinUrl = "https://www.futbin.com/18/playerPrices?player=&all_versions=" + playerIds.join(',')
-
-        var platform = '';
-        if (repositories.User.getCurrent().getSelectedPersona().isPlaystation) platform = "ps";
-        if (repositories.User.getCurrent().getSelectedPersona().isPC) platform = "pc";
-        if (repositories.User.getCurrent().getSelectedPersona().isXbox) platform = "xbox";
-        GM_xmlhttpRequest({
-          method: "GET",
-          url: futbinUrl,
-          onload: function (res) {
-            var futbinData = JSON.parse(res.response);
-            var clubWorth = playerIds.map(function(key) { 
-              if (futbinData[key]) {
-                return parseInt(futbinData[key].prices[platform].LCPrice);
-              } else {
-                return parseInt(0);
-              }
-            }).reduce(function(a, b) { 
-              return a + b; 
-            }, 0);
-            
-            $('.DetailView').prepend(`
-            <div><button class="list" style="cursor: default">
+    gNavManager.onScreenRequest.observe(this, function(obs, event) {
+        if(event === "MyClubSearch") {
+            var currentCount = 0;
+            var currentTotal = 0;
+            setTimeout(function(){
+              $('.MyClubResults').prepend(`
+              <div class="clubValue"><button class="list" style="cursor: default">
               <span class="btn-text">Total Futbin Club Value:</span>
-              <span class="btn-subtext">${clubWorth}</span>
-            </div>`);
-          }
-        });
-      }, 1000);
-    }
-  });
+              <span class="btn-subtext">Calculating, please wait...</span>
+              </div>`);
+            }, 1000);
+            var gettingPlayerData = setInterval(function() {
+                var t = new transferobjects.SearchCriteria();
+                t.type = enums.SearchType.PLAYER;
+                var o = new communication.ClubSearchDelegate(t, currentCount, 90);
+                components.ClickShield.prototype.showShield = function(t) {
```

Comment:
disable-clickshield script does this already. This call would effectively disable the clickshield for the complete app.